packages/core/src/components/popover/popover.tsx

```diff
@@ -375,7 +375,8 @@ export class Popover<
                       onKeyDown: this.handleKeyDown,
                   };
         // Ensure target is focusable if relevant prop enabled
-        const targetTabIndex = openOnTargetFocus && isHoverInteractionKind ? 0 : undefined;
+        const targetTabIndex =
+            content != null && !disabled && openOnTargetFocus && isHoverInteractionKind ? 0 : undefined;
```

Comment:
could you split out and reuse this check?
`const isContentEmpty = content == null || (typeof content === "string" && content.trim() === "");`

Reply:
Okay! I pulled that into a private class method. Let me know if you envisioned a different way to share it between `renderTarget` and `render`. Thanks for the review!